MANIFEST.in

```diff
@@ -1,6 +1,7 @@
 include README.rst
 include CHANGES.rst
 include pyproject.toml
+include conftest.py
```

Comment:
This is for https://github.com/astropy/astropy-integration-testing/pull/26 . If it does not help, I will remove it.